assets/js/modules/adsense/datastore/constants.js

```diff
@@ -26,6 +26,6 @@ export const SITE_STATE_REQUIRES_REVIEW = 'REQUIRES_REVIEW';
 export const SITE_STATE_GETTING_READY = 'GETTING_READY';
 export const SITE_STATE_READY = 'READY';
```

Comment:
See my other comment, let's remove all 4 `SITE_STATE_*` constants in favor of the `API_STATE_*` constants. Everywhere in the code where we currently check for `SITE_STATE_*` constants we can use the corresponding `API_STATE_*` constant instead.

Reply:
Sounds good. Updated!